plumber_rest_api/api/handlers/insert.R

```diff
@@ -50,9 +50,11 @@ handler <- function(data, token, id, bucket){
   }
 
   data_storage$insert(
-    values,
-    insert_time = FALSE,
-    bucket = bucket
+    values$app_name,
+    values$type,
+    values$session,
+    values$details,
+    as.POSIXct(values$time)
```

Comment:
For readability purposes, I think we can name the arguments here. I think the argument names and the names in the value match, but I think it may still be worth to add some argument names after the first two. We were doing this before with `insert_time` and `bucket`.

Reply:
🫡 great point! Done!